cccatalog-api/cccatalog/api/serializers/image_serializers.py

```diff
@@ -31,6 +31,16 @@ class ImageDetailSerializer(ModelSerializer, ImageSerializer):
         required=False,
         help_text="The number of times that an image has been viewed. "
     )
+    attribution = serializers.CharField(
```

Comment:
Could you make this a property of the image model? that way you can set the `source` argument on this field instead of having to insert the value in the view's `get` method.

Reply:
I refactored the code so this is implemented in the model layer. Since the fields are defined as properties, they get mapped to the serializer automatically, so I didn't use the `source` field